src/components/app/ProfileSharing.css

```diff
@@ -61,28 +62,32 @@
   background: var(--internal-uploading-progress-fill-color);
 }
 
-.profileSharingShareButtonButton:not([disabled]) {
+.profileSharingShareButtonButton:not([disabled]),
+.profileSharingSecondShareButtonButton:not([disabled]) {
   background-color: var(--internal-share-button-background-color);
   color: white;
 }
 
 .profileSharingCompositeButtonContainer:not(.currentButtonIsShareButton) > .profileSharingShareButton,
 .profileSharingCompositeButtonContainer:not(.currentButtonIsUploadingButton) > .profileSharingUploadingButton,
 .profileSharingCompositeButtonContainer:not(.currentButtonIsPermalinkButton) > .profileSharingPermalinkButton,
-.profileSharingCompositeButtonContainer:not(.currentButtonIsUploadErrorButton) > .profileSharingUploadErrorButton {
+.profileSharingCompositeButtonContainer:not(.currentButtonIsUploadErrorButton) > .profileSharingUploadErrorButton,
+.profileSharingCompositeButtonContainer:not(.currentButtonIsSecondShareButton) > .profileSharingSecondShareButton {
   pointer-events: none;
 }
 
 .profileSharingShareButtonButton,
 .profileSharingUploadingButtonInner,
 .profileSharingPermalinkButtonButton,
-.profileSharingUploadErrorButtonButton {
-  transition: transform 200ms ease-in-out;
+.profileSharingUploadErrorButtonButton,
+.profileSharingSecondShareButtonButton {
+  transition: transform 200ms ease-in-out, margin-left 200ms ease-in-out;
 }
 
 .profileSharingCompositeButtonContainer:not(.currentButtonIsShareButton) > .profileSharingShareButton > .buttonWithPanelButtonWrapper > .profileSharingShareButtonButton,
 .profileSharingCompositeButtonContainer.currentButtonIsPermalinkButton > .profileSharingUploadingButton > .profileSharingUploadingButtonInner,
-.profileSharingCompositeButtonContainer.currentButtonIsUploadErrorButton > .profileSharingUploadingButton > .profileSharingUploadingButtonInner {
+.profileSharingCompositeButtonContainer.currentButtonIsUploadErrorButton > .profileSharingUploadingButton > .profileSharingUploadingButtonInner,
```

Comment:
Can we simplify this block L89-90?
https://developer.mozilla.org/en-US/docs/Web/CSS/Specificity

Reply:
What do you mean by `move most of this logic to JavaScript` @julienw ?

Reply:
Except this extra share button, there's only 1 button at a time being displayed, and maybe we just shouldn't add them to the DOM when they're not displayed, instead of relying on a bit complex CSS selectors with `:not` and a high specificity (as you said).

Now there's also some transitions going on that we'd like to keep, so maybe that's better to keep it all in CSS. Or maybe `react-transition-group` makes it easy enough to have transitions when adding/removing elements from DOM.

Discussion to have in a separate issue anyway :)

Reply:
@canaltinova Would you like to open a new issue for this? Then I am happy to approve the changes. :)

Reply:
@zoepage Filed issue #1118 :)

Reply:
thx !